front/lib/api/assistant/jit_actions.ts

```diff
@@ -117,13 +120,7 @@ async function getJITActions(
           topK: "auto",
           query: "auto",
           relativeTimeFrame: "auto",
-          dataSources: [
-            {
-              workspaceId: conversation.owner.sId,
-              dataSourceViewId: dataSourceView.sId,
-              filter: { parents: null, tags: null },
-            },
-          ],
+          dataSources,
```

Comment:
This is for retrieval so table unrelated right? How was it working before this for content Nodes?

Reply:
:pray: